doc/admin/env_variables.md

```diff
@@ -68,6 +68,7 @@ Environment variables in this section only apply to the client side.
 |Variable    |Description|
 |------------|-----------|
 |D\_LOG\_FILE|DAOS debug logs (both server and client) are written to `/tmp/daos.log` by default. The debug location can be modified by setting this environment variable ("D\_LOG\_FILE=/tmp/daos_debug.log").|
+|D\_LOG\_SIZE|DAOS debug logs (both server and client) have a 1GB file size limit by default. When this limit is reached, the current log file is closed and renamed with a .old suffix, and a new one is opened. This mechanism will repeat each time the limit is reached, meaning that available saved log records could be found in both ${D_LOG_FILE} and last generation of ${D_LOG_FILE}.old files, to a maximum of the most recent 2*D_LOG_SIZE records.  This can be modified by setting this environment variable ("D_LOG_SIZE=536870912"). Sizes can also be specified in human-readable form using `k`, `m`, `g`, `K`, `M`, and `G`. The lower-case specifiers are base-10 multipliers and the upper case specifiers are base-2 multipliers.|
```

Comment:
we should get out of the habit of putting things on long lines.  The MD parser will remove the new lines

Reply:
Let's fix that in a subsequent doc-only PR.

Reply:
@johannlombardi agreed.   Just making a general comment as it makes reviewing doc changes impossible